wirecard/oxid-ee
# WDPD-103: Language fallback

### This PR

* Adds a prefix to all translation keys of the plugin
* Adapts the scripts for PhraseApp integration to handle prefixed keys
* Overwrites language file with prefixed keys

### How to Test

* Install a language in the shop system which is not available in the plugin or delete german language files in the plugin
* Test with new installed language or german, all translated strings from the plugin should be shown in english

### Jira Links
https://jira.parkside.at/browse/WDPD-103
